lib/queuePopulator/LogReader.js

```diff
@@ -499,6 +499,7 @@ class LogReader {
             const overheadFields = {
                 commitTimestamp: record.timestamp,
                 opTimestamp: entry.timestamp,
+                versionId: entry.overhead && entry.overhead.versionId,
```

Comment:
If this is supported by backbeat...
```suggestion
                versionId: entry.overhead?.versionId,
```

Reply:
it's not :(

Reply:
Why? How can/could we make it supported?

Reply:
To me optional chaining should work as it's supported in node v14+, and I see it in other files

Reply:
Oh my bad, it does work, i remembered it not working a while back.